charts/kong/CHANGELOG.md

```diff
@@ -1,5 +1,25 @@
 # Changelog
 
+## 2.3.0 (not yet released)
+
+### Breaking Changes
+
+* Upgraded CRDs to V1 from the previous deprecated v1beta1.
+  [#391](https://github.com/kong/charts/issues/391)
+  ACTION REQUIRED: This is a breaking change as it makes
+  this chart incompatible with Kubernetes clusters older
+  than v1.16.x. Upgrade your cluster to a version greater
+  than or equal to v1.16 before installing.
+  Note that technically it will remain possible to deploy
+  on older clusters by managing the CRDs manually ahead of
+  time (e.g. intentionally deploying the legacy CRDs) but
+  these configurations will be considered unsupported.
```

Comment:
```suggestion
```

this is a challenging commitment imo - we can't afford to accept the commitment to support unsupported versions "if you just do X and Y"